tests/test_view.py

```diff
@@ -105,3 +146,26 @@ def test_view_share_updates_dictionary_not_overwrite():
     viewclass.share({'test2': 'test2'})
 
     assert viewclass.dictionary == {'test1': 'test1', 'test2': 'test2'}
+
+
+def test_view_cache():
+    """
+    Test for cache template each 5 min
+    """
+
+    container = App()
+
+    view = View().load_container(container)
+
+    container.bind('CacheConfig', cache)
+    container.bind('CacheDiskDriver', CacheDiskDriver)
+    container.bind('CacheManager', CacheManager(container))
+    container.bind('Application', container)
+    container.bind('Cache', container.make('CacheManager').driver('disk'))
+    container.bind('View', view.render)
+
+    view = container.make('View')
+
+    assert view(
```

Comment:
So the test case here which is why there is the bug is the test case should be:

```python
    import time
    assert view(
        'test_cache', {'test': 'test'}
    ).cache_for(1, 'second').rendered_template == 'test'
    
    assert open(glob.glob('bootstrap/cache/test_cache*')[0]).read() == 'test'

    time.sleep(2)

    assert view(
        'test_cache', {'test': 'macho'}
    ).cache_for(5, 'seconds').rendered_template == 'macho'

    assert open(glob.glob('bootstrap/cache/test_cache*')[0]).read() == 'macho'

    time.sleep(2)

    assert view(
        'test_cache', {'test': 'test1'}
    ).cache_for(1, 'second').rendered_template == 'macho'

    assert open(glob.glob('bootstrap/cache/test_cache*')[0]).read() == 'macho'
```

if you get that to work then it should be good to go. `import glob` too. 

The reason is because for some reason it keeps caching the old template over and over again so when I run the test and it works for 1 second, then the cache is expired after the test is ran but if I change the value to macho then it deletes the cache but it puts `test` in the cache instead of `macho`. That test case there should mimic what I'm doing.

that bottom test looks weird because although we are passing in `test1`, we should still get macho back because macho is still in the cache